File: program/SESI4/kondisi4.pas
var
   x : longint;
begin
   readln(x);

   if (x > 0) then begin
      writeln('positif');
   end else if (x = 0) then begin
      writeln('nol');
   end else begin
      // sudah pasti negatif
      writeln('negatif');
   end;
end.

File: topik/pemrograman-dasar-cpp/05-percabangan/asset/kondisi4.cpp
#include <cstdio>

int main() {
  int x;
  scanf("%d", &x);
  if (x > 0) {
    printf("positif\n");
  } else if (x == 0) {
    printf("nol\n");
  } else {
    printf("negatif\n");
  }
}

